### PR TITLE
feat:prizes section

### PR DESCRIPTION
**Pull request by aryab2003**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by Lucif3r-in** [comment]
@aryab2003 You have not imported your component in playground.js.
There are some changes in `yarn.lock` please remove them.

**Comment by aryab2003** [comment]
> @aryab2003 You have not imported your component in playground.js. There are some changes in `yarn.lock` please remove them.
Imported Prizes in playground but then the  deploy preview failed.

File: src/components/Prizes/Prizes.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  TotalPrizes,
+  PrizeText,
+  CashPrize,
+  Swags,
+  Merchandise,
+  PrizesContainerWrapper,
+} from './styles';
+
+// eslint-disable-next-line max-len
+const imageUrls = ["https://res.cloudinary.com/dkpdi1wek/image/upload/v1698839425/gold_zvb6fu.png", "https://res.cloudinary.com/dkpdi1wek/image/upload/v1698839425/silver_ramqno.png", "https://res.cloudinary.com/dkpdi1wek/image/upload/v1698839425/bronze_vlzvme.png"];
```
Comment (Lucif3r-in) [inline src/components/Prizes/Prizes.jsx:16]: Move these to `config/content/Prizes.js`

File: src/components/Prizes/Prizes.jsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+const prizeValues = ['100k', '30k', '20k'];
+const prizeShadowColors = ['#43F4AA', '#02AECC', '#7D34CC'];
```
Comment (Lucif3r-in) [inline src/components/Prizes/Prizes.jsx:18]: Import the colors from `global.css`

**Review by Lucif3r-in — CHANGES_REQUESTED** [review]

**Comment by RickDeb2004** [comment]
@aryab2003 push the recent changes

**Comment by SirSimon162** [comment]
@aryab2003 
- Use image only for trophy image
- Make a component for 'Explore' text
- Make the UI like the design provided
- Fix the height of card when expanded in mobile view

**Comment by aryab2003** [comment]
@SirSimon162 help in box height for mobile design

**Comment by aryab2003** [comment]
@SirSimon162 help me in making a separate winner component

File: src/components/Prizes/styles.js [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    `
+      : ''}
+  @media (max-width: ${smallScreen}) {
+    height: ${(props) => (props.expanded ? '460px' : '350px')};
+  }
+`;
+
+export const PrizesTitle = styled(Heading1)`
+  text-align: center;
+  margin-top: 20px;
```
Comment (SirSimon162) [inline src/components/Prizes/styles.js:102]: Font size should be 40px with line height 40px in PrizesTitle

File: src/components/Prizes/styles.js [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+  margin-top: 16px;
+  @media (max-width: 768px) {
+    font-size: 14px;
+    margin-top: 12px;
+  }
+`;
+
+export const TotalPrizes = styled(Heading1)`
+  font-family: 'Prompt';
+  font-size: 28px;
```
Comment (SirSimon162) [inline src/components/Prizes/styles.js:46]: Font size should be 40px and line height 40px according to the design.

File: src/components/Prizes/styles.js [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+  font-weight: 400;
+  @media (max-width: 768px) {
+    font-size: 14px;
+  }
+`;
+
+export const CashPrize = styled(Body1)`
+  color: var(--color-primary-blue);
+  white-space: nowrap;
+  font-size: 16px;
```
Comment (SirSimon162) [inline src/components/Prizes/styles.js:145]: font size should be 18px and line height 24px.

File: src/components/Prizes/styles.js [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  font-size: 16px;
+  color: #000;
+  visibility: ${(props) => (props.expanded ? 'visible' : 'hidden')};
+  transition: visibility 0.5s ease-in-out;
+  bottom: 20px;
+  margin-top: 16px;
```
Comment (SirSimon162) [inline src/components/Prizes/styles.js:37]: change margin top to 32px for larger screens.

**Comment by SirSimon162** [comment]
Fix the card size more mobile view
image

**Comment by SirSimon162** [comment]
Make it center aligned
image

**Comment by aryab2003** [comment]
> Fix the card size more mobile view 

This issue is not getting fixed at whatever px i am setting..need help

File: src/components/Prizes/styles.js [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+    margin-top: 12px;
+  }
+`;
+
+export const TotalPrizes = styled(Heading1)`
+  font-family: 'Prompt';
+  font-size: 40px;
+  line-height: 40px;
+  font-weight: 600;
+  line-height: 28px;
```
Comment (SirSimon162) [inline src/components/Prizes/styles.js:49]: You have used line height twice. Remove the line with line height 28px.

File: src/components/Prizes/styles.js [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  font-size: 16px;
+  color: #000;
+  visibility: ${(props) => (props.expanded ? 'visible' : 'hidden')};
+  transition: visibility 0.5s ease-in-out;
+  bottom: 20px;
+  margin-top: 32px;
```
Comment (SirSimon162) [inline src/components/Prizes/styles.js:37]: give row-gap: 5px;

**Comment by SirSimon162** [comment]
@aryab2003 For making a separate winner component. Export only trophy image from Figma and put that in a div along with another div containing the Winner text and the icon. Make all of them center aligned.

**Pull request merged**